example/geoextract_example.py

```diff
@@ -2,192 +2,149 @@
 # encoding: utf-8
 
```

Comment:
I'd like to keep the example *really* simple: one file, no external dependencies, just the basics to get you started. From that perspective I still don't see how your refactoring improves the example.